apps/web/src/components/docs/Docs.tsx

```diff
@@ -22,29 +24,69 @@ const getMDXComponent = mdxBundler.getMDXComponent;
 
 const DOCS_WRAPPER_ELEMENT_ID = 'embedded-docs';
 
+const globalOverrides = {
+  frameworkterminal: {
+    FrameworkTerminal: () => <nv-framework-terminal></nv-framework-terminal>,
+  },
+  framework: () => null,
+  echoterminal: {
+    EchoTerminal: () => <nv-framework-terminal></nv-framework-terminal>,
+  },
+};
+
 type DocsProps = PropsWithChildren<
   DocsQueryResults & {
     isLoading?: boolean;
-    actions: ReactNode;
+    actions?: ReactNode;
+    isChildDocs?: boolean;
+    mappings: Record<string, Record<string, string> | string>;
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please add /** */ comment explaining what mappings represents / how callers should use it‏